docs/openstack/mariadb_copy.md

```diff
@@ -83,31 +93,60 @@ SOURCE_MARIADB_IP=127.17.0.100
   ```
   podman run -i --rm --userns=keep-id -u $UID -v $PWD:$PWD:z,rw -w $PWD $MARIADB_IMAGE bash <<EOF
 
-  mysql -h $SOURCE_MARIADB_IP -u root "-p$SOURCE_DB_ROOT_PASSWORD" -N -e 'show databases' | while read dbname; do
-      echo "Exporting \$dbname"
+ # Note we do not want to dump the information and performance schema tables so we filter them
+  mysql -h ${SOURCE_MARIADB_IP} -u root "-p${SOURCE_DB_ROOT_PASSWORD}" -N -e 'show databases' | grep -E -v 'schema|mysql' | while read dbname; do
+      echo "Dumping \${dbname}"
       mysqldump -h $SOURCE_MARIADB_IP -uroot "-p$SOURCE_DB_ROOT_PASSWORD" \
           --single-transaction --complete-insert --skip-lock-tables --lock-tables=0 \
-          --databases "\$dbname" \
-          > "\$dbname".sql
+          "\${dbname}" > "\${dbname}".sql
   done
 
   EOF
   ```
 
-* Before restoring the databases from .sql files, we need to ensure that the neutron
-database name is "neutron" and not "ovs_neutron" by running the command:
-
-  ```
-  sed -i '/^CREATE DATABASE/s/ovs_neutron/neutron/g;/^USE/s/ovs_neutron/neutron/g' ovs_neutron.sql
-  ```
-
 * Restore the databases from .sql files into the podified MariaDB:
 
   ```
-  for dbname in cinder glance keystone nova_api nova_cell0 nova ovs_neutron placement; do
-      echo "Importing $dbname"
-      oc run mariadb-client --image $MARIADB_IMAGE -i --rm --restart=Never -- \
-         mysql -h "$PODIFIED_MARIADB_IP" -uroot "-p$PODIFIED_DB_ROOT_PASSWORD" < "$dbname.sql"
+  # db schemas to rename on import
+  declare -A db_name_map
+  db_name_map["nova"]="nova_cell1"
```

Comment:
I'm wondering what will be the naming scheme in the source DB if there are multiple cells there.

---
Also we need to note somewhere to the end user to make sure that if there is a cell db adopted with the name as nova_cell1 then there is a cellTemplate in the OpenStackControlPlane CR named "cell1".

Reply:
what if there is already nova and nova_cell1 in the source DBs? That invalidates the mapping and nova_cell1 db will be lost?

Reply:
@stuggi if you recall tripleo cells DB naming scheme for more than >1 cell perchance?